sdk/identity/azure-identity/azure/identity/_credentials/cli_credential.py

```diff
@@ -0,0 +1,66 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import sys
+_IS_PY2 = sys.version_info[0] == 2
+
+import platform
+_USE_SHELL = platform.system() == 'Windows'
+
+import json
+import time
+from datetime import datetime
+
+from subprocess import check_output, CalledProcessError, STDOUT
+
+from azure.core.credentials import AccessToken
+from azure.core.exceptions import ClientAuthenticationError
+
+class CliCredential(object):
+
+    _LINUX_CMD_NOT_FOUND = 'command not found'
+    _WINDOWS_CMD_NOT_FOUND = 'is not recognized as'
+    _DEFAULT_PREFIX = "/.default"
+    _CLI_NOT_INSTALLED_ERR = "Azure CLI not installed"
+
+    def get_token(self, *scopes, **kwargs): # pylint:disable=unused-argument
+        command = ['az', 'account', 'get-access-token']
+
+        if scopes:
+            resource = scopes[0]
+            if resource.endswith(self._DEFAULT_PREFIX):
+                resource = resource[:-len(self._DEFAULT_PREFIX)]
```

Comment:
Is there a utility method in azure.core.credentials to do the scope to resource conversion? If there isn't perhaps we should create a work item to add it? cc: @chlowell

Reply:
No there isn't.

Reply:
Can I resolve this conversation? @chlowell  @schaabs